kotlin/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM iron/java
+
+ENV VERSION 1.0.5
+
+ENV KOTLIN_HOME=/usr/share/kotlin
+ENV KOTLIN_COMPILER_URL https://github.com/JetBrains/kotlin/releases/download/v${VERSION}/kotlin-compiler-${VERSION}.zip
```

Comment:
Thanks. I didn't realise multiple layers would be impacted by this. The KOTLIN_HOME wasn't used but VERSION is required for the other line so I've combined them into one.